src/js/containers/ImageOptimizationCard/ImageOptimizationCard.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import _ from 'lodash';
+import { asyncPluginUpdateSetting } from '../../actions/pluginSettings';
+import { getPluginSettingsValueForZoneId } from '../../selectors/pluginSettings';
+import { Card, CardSection, CardContent, CardDrawers } from 'cf-component-card';
+import CustomCardControl from '../../components/CustomCardControl/CustomCardControl';
+import { asyncZoneUpdateSetting } from '../../actions/zoneSettings';
+import Toggle from 'cf-component-toggle';
+
+const SETTING_NAME_MIRAGE = "mirage";
+const SETTING_NAME_POLISH = "polish";
+const MINIMUM_PLAN = "Pro Plan";
+
+class ImageOptimizationCard extends Component {
+
+    handleChange(value) {
+        let { activeZoneId, dispatch } = this.props;
+        value = (value === true ? "on" : "off");
+        dispatch(asyncZoneUpdateSetting(SETTING_NAME_MIRAGE, activeZoneId, value));
+        dispatch(asyncZoneUpdateSetting(SETTING_NAME_POLISH, activeZoneId, value));
+    }
+
+    render() {
+    	let { activeZone, config, zones } = this.props;
+    	let zone;
+        if(!_.isEmpty(activeZone.name)) {
```

Comment:
Why are you checking if this is empty?  This card should never render without an `activeZone`.